src/sentry/userfeedback/models.py

```diff
@@ -0,0 +1,30 @@
+from django.db import models
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import (
+    BoundedBigIntegerField,
+    DefaultFieldsModel,
+    region_silo_only_model,
+    sane_repr,
+)
+from sentry.db.models.fields.array import ArrayField
+
+
+@region_silo_only_model
+class UserFeedback(DefaultFieldsModel):
+    __relocation_scope__ = RelocationScope.Excluded
+
+    project_id = BoundedBigIntegerField()
+    replay_id = models.CharField(max_length=32, null=True)
+    url = models.CharField(max_length=1000, null=True)
+    error_ids = ArrayField()
+    trace_ids = ArrayField()
+    feedback_text = models.TextField()
+    context = models.JSONField(null=True)
```

Comment:
Can we verify that this produces a JSONB column type (as opposed to the JSON column type)?  You should be able to run the migration and inspect the table's schema.

Reply:
<img width="905" alt="SCR-20230906-mrep" src="https://github.com/getsentry/sentry/assets/56095982/c298a091-e5ab-45ea-b09e-878979eeeae3">